traffic_ops/traffic_ops_golang/federation_resolvers/federation_resolvers.go

```diff
@@ -136,13 +135,28 @@ func Read(w http.ResponseWriter, r *http.Request) {
 	} else {
 		log.Warnf("Couldn't get config %v", e)
 	}
+
+	// Based on version we load types - for version 5 and above we use FederationResolverV5
+	var resolvers []interface{}
+	if inf.Version != nil && inf.Version.Major >= 5 && inf.Version.Minor >= 0 {
```

Comment:
This is fine, but there is a utility function that can be used in place of this if statement like this:
`if inf.Version.GreaterThanOrEqualTo(&api.Version{Major: 5, Minor: 0})`